src/models/gaze17/internal-overview.md

```diff
@@ -0,0 +1,27 @@
+# Gazelle (Internal Overview)
+
+## Bottom components and connectors:
+
+![Internal Components](./img/components-highlighted.webp)
+
+- Webcam/microphone connector is highlighted in pink
+- Fan connectors are highlighted in orange
+- LCD panel connector is highlighted in green
+- Wireless card is highlighted in blue
+- RAM is highlighted in red
+- M.2 SSDs are highlighted in yellow
```

Comment:
The drive slots have different spec support for NVMe gen 3/SATA and just NVMe gen4 support and should be called out on this overview

Reply:
We have already had this conversation. https://github.com/system76/tech-docs/pull/87#discussion_r676792485

Reply:
Addressed with 80a96d06d75277efaecd488944d89559edc2c7ba even though I disagree with the need for this.